src/disease_model/requirements.txt

```diff
@@ -1 +1,4 @@
-pandas
+pandas~=0.25.3
+gitpython~=3.1.2
+numpy~=1.17.5

```

Comment:
This might happen in the future as well, should we have a single requirements.txt file then for the whole package, I might be wrong here, what's the benefit of having separate requirements.txt

Reply:
I agree, supposedly it was exactly that they wouldn't interfere. I don't see how that's possible without using containers, we should bring this up in the meeting

Reply:
I agree, supposedly it was exactly that they wouldn't interfere. I don't see how that's possible without using containers, we should bring this up in the meeting

Reply:
I agree, supposedly it was exactly that they wouldn't interfere. I don't see how that's possible without using containers, we should bring this up in the meeting

Reply:
I agree, supposedly it was exactly that they wouldn't interfere. I don't see how that's possible without using containers, we should bring this up in the meeting

Reply:
I agree, supposedly it was exactly that they wouldn't interfere. I don't see how that's possible without using containers, we should bring this up in the meeting